config/jobs/ppc64le-cloud/builds/kubernetes-postsubmit.yaml

```diff
@@ -44,3 +44,94 @@ postsubmits:
+    - name: postsubmit-master-golang-kubernetes-conformance-test-ppc64le
+      cluster: k8s-ppc64le-cluster
+      decorate: true
+      run_if_changed: '^kubernetes/master/golang/master/'
+      interval: 1h
+      extra_refs:
+        - base_ref: master
+          org: ppc64le-cloud
+          repo: kubetest2-plugins
+          workdir: true
+        - base_ref: v1.10.0
+          org: IBM-Cloud
+          repo: terraform-provider-ibm
+      spec:
+        volumes:
+          - name: powercloud-bot-key
+            secret:
+              defaultMode: 256
+              secretName: bot-ssh-secret
+        containers:
+          - image: quay.io/powercloud/all-in-one:0.1
+            command:
+              - /bin/bash
+            volumeMounts:
+              - mountPath: /etc/secret-volume
+                name: powercloud-bot-key
+                readOnly: true
+            envFrom:
+              - secretRef:
+                  name: ibm-cloud-credentials
+            args:
+              - -c
+              - |
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+                set -o xtrace
+                export PATH=$GOPATH/bin:$PATH
+                export GO111MODULE=on
+
+                go install ./...
+
+                mkdir -p $HOME/.terraform.d/plugins/linux_`go env GOARCH`
+
+                pushd $GOPATH/src/github.com/IBM-Cloud/terraform-provider-ibm
+                go build .
+                cp terraform-provider-ibm $HOME/.terraform.d/plugins/linux_`go env GOARCH`/terraform-provider-ibm_v1.10.0
+                popd
+
+                go get github.com/hashicorp/terraform-provider-null
+                cp $GOPATH/bin/terraform-provider-null $HOME/.terraform.d/plugins/linux_`go env GOARCH`/
+
+                go get sigs.k8s.io/kubetest2@latest
+                go get sigs.k8s.io/kubetest2/kubetest2-tester-exec@latest
+
+                TIMESTAMP=$(date +%s)
+
+                # Details of IBM S3 storage
+                S3_SERVER=s3.us-south.cloud-object-storage.appdomain.cloud
+                BUCKET=ci-builds
+                DIRECTORY=kubernetes/master/golang/master
+                build_commit=$(jq -r .commit $GOPATH/src/github.com/ppc64le-cloud/builds/kubernetes/master/golang/master/build.yaml)
+
+                K8S_BUILD_VERSION=$build_commit
+                # e2e.test binary needed for running tests using k8s tests bits from IBM s3 storage
+                URL=https://$S3_SERVER/$BUCKET/$DIRECTORY/$K8S_BUILD_VERSION
+                wget $URL/kubernetes-test-linux-ppc64le.tar.gz -P /usr/local/bin
+                tar -zxf /usr/local/bin/kubernetes-test-linux-ppc64le.tar.gz --directory /usr/local/bin
+                cp /usr/local/bin/kubernetes/test/bin/e2e.test /usr/local/bin/kubernetes/test/bin/ginkgo /usr/local/bin
```

Comment:
this code can be replaced by:

```
wget -qO- $URL/kubernetes-test-linux-ppc64le.tar.gz | tar xfz kubernetes-test-linux-ppc64le.tar.gz --strip-components=3 -C /usr/local/bin
```

This will help you save some diskspace